htdocs/image.php

```diff
@@ -363,8 +366,8 @@ function imageFilenameCheck($imageUrl)
  * Get/check editing parameters
  */
 // width, height
-$max_width = isset($_GET['width']) ? (int)$_GET['width'] : false;
-$max_height = isset($_GET['height']) ? (int)$_GET['height'] : false;
+$max_width = $width = Request::getInt('width', 0, 'GET');
+$max_height = $height = Request::getInt('height', 0, 'GET');
```

Comment:
why not:
$max_width = Request::getInt('width', 0, 'GET');
$max_height = Request::getInt('height', 0, 'GET');